app/src/main/java/com/aaronjwood/portauthority/network/Wireless.java

```diff
@@ -186,6 +186,38 @@ public int getInternalWifiSubnet() {
         return netmask;
     }
 
+
+    public int getInternalWifiIpAddressAsInt() {
```

Comment:
Also, another good reason to have this generic is to use it to return `byte` for this fix for the host scan in `ScanHostsRunnable`:
```
byte[] bytes = BigInteger.valueOf(i).toByteArray();
socket.connect(new InetSocketAddress(InetAddress.getByAddress(bytes), 7), 150);
```